README.adoc

```diff
@@ -252,6 +252,12 @@ This behavior is controlled by the `spring.cloud.kubernetes.config.paths` proper
 addition to or instead of the mechanism described earlier.
 Multiple (exact) file paths can be specified in `spring.cloud.kubernetes.config.paths` by using the `,` delimiter
 
+**Notes:**
+`spring.cloud.kubernetes.config.paths` property processing behavior was changed in 1.0.0 version of
```

Comment:
Prior versions were pre-release software and not officially supported. So even though the behavior changed, this is the GA behavior. I think the not should say 
```
You have to provide full exact path to each property file, because directories are not being recursively parsed.
```
Drop the "as it was before". 

That's the kind of thing for release notes.

Reply:
Ok, I've just thought about it too, but decided to make it clear for people who upgraded from previous versions.